Source/Core/DolphinQt2/GameList/GameList.cpp

```diff
@@ -126,18 +172,191 @@ void GameList::OpenProperties()
   properties->show();
 }
 
+void GameList::ExportWiiSaves()
+{
+  QMessageBox result_dialog(this);
+  if (GameFile(GetSelectedGame()).ExportWiiSaves())
+  {
+    result_dialog.setIcon(QMessageBox::Information);
+    result_dialog.setText(tr("Successfully exported save files"));
+  }
+  else
+  {
+    result_dialog.setIcon(QMessageBox::Critical);
+    result_dialog.setText(tr("Failed to export save files!"));
+  }
+  result_dialog.exec();
+}
+
 void GameList::OpenWiki()
 {
   QString game_id = GameFile(GetSelectedGame()).GetGameID();
   QString url = QStringLiteral("https://wiki.dolphin-emu.org/index.php?title=").append(game_id);
   QDesktopServices::openUrl(QUrl(url));
 }
 
+void GameList::CompressISO()
+{
+  const auto original_path = GetSelectedGame();
+  auto file = GameFile(original_path);
+
+  const bool compressed = (file.GetBlobType() == DiscIO::BlobType::GCZ);
+
+  QString label_text = tr("Select where you want to save the ");
+  QString file_types;
+  QString default_file = QFileInfo(GetSelectedGame()).dir().absoluteFilePath(file.GetGameID());
+
+  if (compressed)
+  {
+    label_text.append(tr("decompressed image"));
+    default_file.append(QStringLiteral(".iso"));
+    file_types = tr("Uncompressed GC/Wii image (*.iso *.gcm)");
+  }
+  else
+  {
+    label_text.append(tr("compressed image"));
+    default_file.append(QStringLiteral(".gcz"));
+    file_types = tr("Compressed GC/Wii images (*.gcz)");
+
+    if (file.GetPlatformID() == DiscIO::Platform::WII_DISC)
+    {
+      QMessageBox wii_warning(this);
+      wii_warning.setIcon(QMessageBox::Warning);
+      wii_warning.setText(tr("Are you sure?"));
+      wii_warning.setInformativeText(tr("Compressing Wii ISOs is an irreversible process. Your "
+                                        "image will keep working as usual."));
```

Comment:
Could you copy the wording from DolphinWX instead of coming up with something new? We went through a lot of bikeshedding to get something that everyone was happy with :(

(Well, at least I assume we did... It would be strange if the PR that added it would've been bikeshed-free when it came to the wording.)